main/reference/vstorage-ref.md

```diff
@@ -0,0 +1,434 @@
+      'psm-IST-DAI_axl': Object @Alleged: InstanceHandle#board01867 {},
+      'psm-IST-DAI_grv': Object @Alleged: InstanceHandle#board02568 {},
+      'psm-IST-USDC_axl': Object @Alleged: InstanceHandle#board05669 {},
+      'psm-IST-USDC_grv': Object @Alleged: InstanceHandle#board05970 {},
+      'psm-IST-USDT_axl': Object @Alleged: InstanceHandle#board02271 {},
+      'psm-IST-USDT_grv': Object @Alleged: InstanceHandle#board01272 {},
+      reserve: Object @Alleged: InstanceHandle#board06458 {},
+      reserveGovernor: Object @Alleged: InstanceHandle#board03365 {},
+      'scaledPriceAuthority-stATOM': Object @Alleged: InstanceHandle#board05892 {},
+      'stATOM-USD price feed': Object @Alleged: InstanceHandle#board04091 {},
+      walletFactory: Object @Alleged: InstanceHandle#board06366 {},
+    }
+```
+
+## vstorage: well-known assets
+
+`published.agoricNames.issuer` has Issuers of well-known assets.
+
+```js
+    {
+      ATOM: Object @Alleged: ATOM issuer#board02656 {},
+      BLD: Object @Alleged: BLD issuer#board0592 {},
+      DAI_axl: Object @Alleged: DAI_axl issuer#board02437 {},
+      DAI_grv: Object @Alleged: DAI_grv issuer#board05039 {},
+      IST: Object @Alleged: IST issuer#board0223 {},
+      Invitation: Object @Alleged: Zoe Invitation issuer#board0371 {},
+      KREAdCHARACTER: Object @Alleged: KREAdCHARACTER issuer#board01386 {},
+      KREAdITEM: Object @Alleged: KREAdITEM issuer#board03687 {},
+      USDC_axl: Object @Alleged: USDC_axl issuer#board05141 {},
+      USDC_grv: Object @Alleged: USDC_grv issuer#board00443 {},
+      USDT_axl: Object @Alleged: USDT_axl issuer#board06445 {},
+      USDT_grv: Object @Alleged: USDT_grv issuer#board01547 {},
+      stATOM: Object @Alleged: stATOM issuer#board00689 {},
+    }
+```
+
+`published.agoricNames.brand` has the well-known Brands.
+
+```js
+    {
+      ATOM: Object @Alleged: ATOM brand#board05557 {},
+      BLD: Object @Alleged: BLD brand#board0566 {},
+      DAI_axl: Object @Alleged: DAI_axl brand#board05736 {},
+      DAI_grv: Object @Alleged: DAI_grv brand#board03138 {},
+      IST: Object @Alleged: IST brand#board0257 {},
+      Invitation: Object @Alleged: Zoe Invitation brand#board0074 {},
+      KREAdCHARACTER: Object @Alleged: KREAdCHARACTER brand#board03281 {},
+      KREAdITEM: Object @Alleged: KREAdITEM brand#board00282 {},
+      USDC_axl: Object @Alleged: USDC_axl brand#board03040 {},
+      USDC_grv: Object @Alleged: USDC_grv brand#board04542 {},
+      USDT_axl: Object @Alleged: USDT_axl brand#board01744 {},
+      USDT_grv: Object @Alleged: USDT_grv brand#board03446 {},
+      stATOM: Object @Alleged: stATOM brand#board00990 {},
+      timer: Object @Alleged: timerBrand#board0425 {},
+    }
+```
+
+`published.agoricNames.oracleBrand` has the well-known oracle brands.
+
+    {
+      ATOM: Object @Alleged: Brand#board03935 {},
+      USD: Object @Alleged: Brand#board01034 {},
+      stATOM: Object @Alleged: Brand#board04388 {},
+    }
```

Comment:
missed the triple-quotes here.

Reply:
good thing you're reading more carefully than I'm writing :)